python-template/{{cookiecutter.placeholder_repo_name}}/setup.py

```diff
@@ -36,17 +36,43 @@ def load_requirements(*requirements_paths):
     with -c in the requirements files.
     Returns a list of requirement strings.
     """
+    by_canonical_name = {}
```

Comment:
A comment here with an example entry would be helpful. It took me a minute to figure out how it was being used.

Reply:
Sure, maybe something like this:

```python
    # e.g. {"django": "Django", "openedx-events": "openedx_events"}
```